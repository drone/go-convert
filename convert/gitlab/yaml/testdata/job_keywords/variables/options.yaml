# https://docs.gitlab.com/ee/ci/yaml/#variablesoptions

variables:
  DEPLOY_ENVIRONMENT:
    value: "staging"
    options:
      - "production"
      - "staging"
      - "canary"
    description: "The deployment target. Set to 'staging' by default."