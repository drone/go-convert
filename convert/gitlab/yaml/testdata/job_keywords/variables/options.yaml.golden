variables:
    DEPLOY_ENVIRONMENT:
        value: staging
        description: The deployment target. Set to 'staging' by default.
        options:
            - production
            - staging
            - canary
        expand: true
